nmpolicy/internal/lexer/rune.go

```diff
@@ -31,3 +32,7 @@ func (l *lexer) isSpace() bool {
 func (l *lexer) isEOF() bool {
 	return l.scn.Rune() == scanner.EOF
 }
+
+func (l *lexer) isString() bool {
+	return strings.ContainsRune(`"'`, l.scn.Rune())
```

Comment:
As far as I know, yamls support strings with no quotes at all. Do we want to block it for nmpolicy?

Reply:
This is not yaml, this is the capture expressions.